subcommands/git/cmd.go

```diff
@@ -73,10 +74,16 @@ func doGitCreds(cmd *cobra.Command, args []string) {
 	helperName := "fio"
 	dst := filepath.Join(helperPath, GIT_CREDS_HELPER)
 	if runtime.GOOS == "windows" {
+		// To get around edge cases with git on Windows we use the absolute path
+		// So for example the following path will be used: C:/Program\\ Files/Git/bin/git-credential-fio.exe
 		dst += ".exe"
-		helperName += ".exe"
+		helperName = filepath.ToSlash(dst)
+	} else if credsPath.Changed {
+		helperName = dst
 	}
 
+	helperName = subcommands.EncodeWhiteSpace(helperName)
```

Comment:
I'm now confused by this whole PR. I think the normal case (not windows i guess) is to:
 * find a directory in `$PATH` we can write to
 * link file fioctl -> git-credential-fio
 * run `git config ... ` to set the helper

The whole point of finding something in $PATH is so we don't need absolute paths?

It almost sounds like on windows - we aren't finding something in the user's actual path?

If we do need absolute paths, rather than trying to "encode" (nitpick - I think we are *escaping*) - we probably need to put quotes around what we present to git-config. That way what winds up in git config looks like:
```
helper = "c:\\program files\blah.exe"
```
I suspect the quotations would address things beyond spaces we haven't considered.

That said - seems like the bug is that we can't find PATH correctly on windows

Reply:
I agree, the problem partly stems from us requiring the user to run as `root`/`Administrator` and the `PATHS` being different for them on Windows. I will briefly look into a way if you can _easily_ get the original calling user on Windows. If that is the case we could either add in logic to add things to the PATH for that user, or place the symlink into one of the places in their PATH locations. Then we can drop this special case for Windows again, apart from the naming having an extra `.exe`. :+1: